core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1525,7 +1525,7 @@ public String toString() {
           .build();
   public static final PropertyKey MASTER_WORKER_HEARTBEAT_INTERVAL =
       new Builder(Name.MASTER_WORKER_HEARTBEAT_INTERVAL)
-          .setDefaultValue("10sec")
+          .setDefaultValue("1sec")
           .setDescription("The interval between Alluxio master and worker heartbeats.")
```

Comment:
can we update this description to be more accurate? It is quite confusing to have both parameters to represent different concepts.

Reply:
Actually the biggest difference between these two properties is `MASTER_WOREKR_HEARTBEAT_INTERVAL` is used by AlluxioMaster. `WORKER_BLOCK_HEARTBEAT_INTERVAL` is used by AlluxioWorker.
What they actually do is not that important
Will update the description